.github/workflows/continuous_integration.yml

```diff
@@ -35,6 +35,34 @@ jobs:
 
       - name: Update
         run: ./scripts/update
+        env:
+          PUBLIC_URL: https://azavea.github.io/green-equity-demo/
 
       - name: Test
         run: ./scripts/test
+
+      - name: Upload site artifact
+        if: github.event_name == 'push'
+        uses: actions/upload-pages-artifact@v1
+        with:
+          path: "src/app/build"
+
+  deploy:
+    name: deploy
+    needs: build
+    runs-on: ubuntu-latest
+    if: github.event_name == 'push'
```

Comment:
Does this mean a deploy happens on every push event? Would be good to have it only happen on pushes to develop.

Reply:
Ah, is it because `push` is distinct from `pull_request`? I see now.

Reply:
Yep.`push` is also limited to branches that match the patterns at the top of the file (`develop`, `test/**`, etc...).